index.html

```diff
@@ -20,377 +20,273 @@
+        <div class="frame2">
+          <div class="top-frame2">
+            <div class="rectangle"></div>
+            <div class="framework">
+              <h4>Frameworks</h4>
+            </div>
           </div>
-        </div>
-        <div class="inner-frame">
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        </div>
-      </div>
-      <div class="framework">
-        <div class="sub-framework">
-          <div class="square"></div>
-          <div class="frame">
-            <h4>Frameworks</h4>
+          <div class="frame-content">
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RsPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
           </div>
+
         </div>
-        <div class="inner-framework">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-      </div>
-      <div class="main-skill">
-        <div class="sub-skills">
-          <div class="circle"></div>
-          <div class="skills">
-            <h4>Skills</h4>
+
+        <div class="frame3">
+          <div class="top-frame3">
+            <div class="circle"></div>
+            <div class="skills">
+              <h4>skills</h4>
+            </div>
           </div>
-        </div>
-        <div class="inner-skills">
-          <ul>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+          <div class="skill-content">
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+
         </div>
       </div>
-      
-
-      
-      
-
     </section>
 
     <section class="contact">
       <div class="contact-text">
-        <p>
-          I'm always interested in hearing about new projects, 
+        <p>I'm always interested in hearing about new projects, 
           so if you'd like to chat please get in touch.
         </p>
       </div>
+      <form action="https://formspree.io/f/mnqreddp" method="post" class="form">
+        <div class="form-row">
+          <input type="text" class="form-text" maxlength="30" placeholder="first name" required>
+        <input type="text"  class="form-text" maxlength="30" placeholder="last name" required>
+        </div>
+        <div class="form-mail">
+          <input type="email"  class="contact-form-email" placeholder="Email address" required>
+        </div>
+          <textarea name="contact"  class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
```

Comment:
- [ ] As per the project requirement you should stick to the design as much as possible. Please remove the border of the text area so it matches the design. See the image below for reference
### Your design
![yourrrrrrrrrrrrr](https://user-images.githubusercontent.com/81727988/206646867-f06761ae-5940-4d45-9758-838b895de14b.png)

### Figma design
![yourrrrr](https://user-images.githubusercontent.com/81727988/206646858-2f5469eb-399c-44c7-98f5-aa047682eee1.png)